components/components/view-modes/highlighted-small/_node-highlighted-small.css

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file
+ * This file is used to style the 'highlighted-small' view mode.
+ */
+
+/*
+  Highlighted Small
+
+  .node--view-mode-highlighted-small - This presents the 'Highlighted Small' view mode.
+
+  Markup: node--highlighted-small.html.twig
+
+  Style guide: components.view_modes.highlighted_small
+*/
+
+.node--view-mode-highlighted-small {
+  border: 1px solid #fcece7;
+}
+
+.node--view-mode-highlighted-small .node__meta {
+  padding: 1.5rem;
+}
+
+.node--view-mode-highlighted-small .node__meta > h2 {
+  margin-bottom: 1rem;
+}
+
+.node--view-mode-highlighted-small .node__meta > h2 .field--name-title {
+  font-family: 'Scope One';
+  font-size: 1.424rem;
+}
+
+.node--view-mode-highlighted-small .node__meta > h2 a {
+  color: #000000;
```

Comment:
It is better to use the shorter version for hex colors: #000

> When hex values are used for colors, use lowercase and, if possible, the shorthand syntax, e.g. #aaa.

https://www.drupal.org/docs/develop/standards/css/css-formatting-guidelines#properties

Reply:
Well spotted. I didn't know that was a Drupal policy.